### PR TITLE
[quest] ADDED: 'Molten Blast' Shaman Quests To QuestDB

### LINKED ISSUE #5607: [quest] 'Molten Blast' Shaman Rune Quest Missing From QuestDB
## Description

The SoD Shaman rune quest 'Molten Blast' is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/guide/season-of-discovery/shaman-class-runes-discovery-location
https://www.wowhead.com/classic/spell=425344/engrave-gloves-molten-blast

## Screenshots

N/a

## Questie & WoW version

v9.2.17
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #5607
Linked To #5443

## Proposed changes

- ADDED: All 'Molten Blast' Shaman fake rune quests and they are now in the QuestDB. The Durotar location is accessible to users, however the Mulgore location is not currently due to Bael'dun Dwarf spawns.

### CODE REVIEW

File: Database/Corrections/sodQuestFixes.lua [hunk 1]
```diff
@@ -3168,6 +3168,30 @@ function SeasonOfDiscovery:LoadQuests()
...
+            [questKeys.questLevel] = 9,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Use fire spells to get 5 stacks of debuff on the Frozen Makura then loot it and equip the Icon and follow its instructions to receive the rune."},
+            [questKeys.requiredSpell] = -425344,
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
+        [90217] = {
+            [questKeys.name] = "Molten Blast",
+            [questKeys.startedBy] = {{2989,2990},{404352}},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3185]: Let's do the same as with other quests and also remove the NPCs from this. The object location + the description text should be enough 👍🏻 

```suggestion
            [questKeys.startedBy] = {nil,{404352}},
```

File: Database/Corrections/SeasonOfDiscovery.lua [hunk 2]
```diff
@@ -361,6 +363,7 @@ local questsToBlacklistBySoDPhase = {
         [90200] = true, -- Hiding Rogue Mutilate Dun Morogh for now as there are too many icons
         [90201] = true, -- Hiding Rogue Shiv Duskwood for now as there are too many icons
         [90204] = true, -- Hiding Shaman Way of Earth Silverpine Forest for now as there are too many icons
+        [90217] = true, -- Hiding Shaman Molten Blast Mulgore for now as there are too many icons
```
Comment (BreakBB) [inline Database/Corrections/SeasonOfDiscovery.lua:366]:
```suggestion
```

File: Database/Corrections/sodQuestFixes.lua [hunk 3]
```diff
@@ -3168,6 +3168,30 @@ function SeasonOfDiscovery:LoadQuests()
...
         },
+        [90216] = {
+            [questKeys.name] = "Molten Blast",
+            [questKeys.startedBy] = {{208275}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 9,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.SHAMAN,
+            [questKeys.objectivesText] = {"Use fire spells to get 5 stacks of debuff on the Frozen Makura then loot it and equip the Icon and follow its instructions to receive the rune."},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:3179]: I don't think you can do this rune on your own, just like the other runes where you need to attack some frozen NPC with fire spells.

For the "Burnout" rune we (yeah, me) added the following text:

> Gather some Mage and Warlock friends and attack the Frozen Makrura with fire spells.

I am open for suggestions, but should streamline the wording then I guess.

**Review by BreakBB — CHANGES_REQUESTED** [review]

File: Database/Corrections/SeasonOfDiscovery.lua (hunk 2 above)
Comment (Sommos) [reply Database/Corrections/SeasonOfDiscovery.lua]: Fixed in: https://github.com/Questie/Questie/pull/5611/commits/41d028fb6a8d634631939db7bb3efe53f6abc5ef

File: Database/Corrections/sodQuestFixes.lua (hunk 1 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Fixed in: https://github.com/Questie/Questie/pull/5611/commits/41d028fb6a8d634631939db7bb3efe53f6abc5ef

File: Database/Corrections/sodQuestFixes.lua (hunk 3 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Fixed in: https://github.com/Questie/Questie/pull/5611/commits/41d028fb6a8d634631939db7bb3efe53f6abc5ef

I think it's okay to have this as the line right now, but maybe we should adjust it based on user feedback?

**Review by BreakBB — APPROVED** [review]

**Pull request merged**